packages/builder/src/components/design/settings/controls/DataSourceSelect/DataSourceCategory.svelte

```diff
@@ -29,7 +29,7 @@
       on:click={() => onSelect(data)}
     >
       <span class="spectrum-Menu-itemLabel">
-        {data.label}
+        {data.datasource?.name ? `${data.datasource.name} -` : ""} {data.label}
```

Comment:
```suggestion
        {data.datasource?.name ? `${data.datasource.name} - ` : ""}{data.label}
```
NAB of course but keeps my OCD happy by not including a space if it's not needed. Sorry 😂